core/src/components/datetime/datetime.ios.scss

```diff
@@ -72,6 +72,120 @@
   text-transform: uppercase;
 }
 
+@supports (border-radius: mod(1px, 1px)) {
+  .calendar-days-of-week .day-of-week {
+    /**
+     * When the computed font size is 24px
+     * we want the days of the week to only
+     * show the first letter of each day.
+     *
+     * We use the modulus operator to ensure
+     * that any font size less than 24px is
+     * unchanged (i.e. 16 % 24 = 16). If the
+     * font size is 24px then the remainder
+     * will be 0.
+     */
+    $max-font-size: min(1rem, $datetime-dynamic-font-breakpoint);
+    $remainder: mod($max-font-size, $datetime-dynamic-font-breakpoint);
+
+    /**
+     * From here, we scale the remainder to get
+     * a width value larger than 20px. The purpose of
+     * scaling the remainder is to get a border radius
+     * large enough such that the element resembles a circle.
```

Comment:
I think this is in the wrong place? This part is calculating the font size of the weekday labels, not the border radius. Maybe I'm misunderstanding?

Reply:
You are correct - that comment is in the wrong place. Fixed in You are correct - that comment is in the wrong place. Fixed in https://github.com/ionic-team/ionic-framework/pull/28129/commits/67b2c9c1fe4c2fd1a298da109cf876dddf49d356